filetracker/servers/run.py

```diff
@@ -123,6 +128,21 @@ def main(args=None):
         os.unlink(conf_path)
 
 
+def db_init(db_dir):
+    mkdir(db_dir)
+    db_env = bsddb3.db.DBEnv()
+    db_env.open(
+            db_dir,
+            bsddb3.db.DB_CREATE
+            | bsddb3.db.DB_INIT_LOCK
+            | bsddb3.db.DB_INIT_LOG
+            | bsddb3.db.DB_INIT_MPOOL
+            | bsddb3.db.DB_INIT_TXN
+            | bsddb3.db.DB_REGISTER
+            | bsddb3.db.DB_RECOVER)
+    db_env.close()
```

Comment:
I think it would be good to add a log entry before opening and after opening, so that it's clear in the logs that the opening process, incl. recovery, is taking place.

Reply:
Done.